R/pkg/R/DataFrame.R

```diff
@@ -1137,11 +1137,22 @@ setMethod("summarize",
 #' @rdname dapply
 #' @name dapply
 #' @export
+#' @examples
+#' \dontrun{
+#'   df <- createDataFrame (sqlContext, mtcars)
+#'   df1 <- dapply(df, function(x) { x }, schema(df))
```

Comment:
could we have an more elaborate example to explain how `func` should expect or handle "each partition of the DataFrame"?